CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and [multi-factor authentication](https://docs.mattermost.com/deployment/auth.html) is enabled, ensure your users have upgraded to mobile app version 1.17 or later. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
```

Comment:
Not necessary to highlight these as there is no change to the user

Reply:
1.17 was mainly just bug fixes. No highlights required

Reply:
Elias said that these should be highlighted. @enahum What do you think?

Reply:
The Share Extension was re-done from the ground up, so I do think it should be highlighted. the upgrade to RN 0.58.5 is of no importance

Reply:
ya it was rebuilt but there's no change to the end user right? So why highlight it?